moto/ecr/models.py

```diff
@@ -245,6 +256,38 @@ def put_image(self, repository_name, image_manifest, image_tag):
         repository.images.append(image)
         return image
 
+    def batch_get_image(self, repository_name, registry_id=None, image_ids=None, accepted_mdeia_types=None):
+        if repository_name in self.repositories:
+            repository = self.repositories[repository_name]
+        else:
+            raise RepositoryNotFoundException(repository_name, registry_id or DEFAULT_REGISTRY_ID)
+
+        if image_ids:
+            response = set()
+            for image_id in image_ids:
+                found = False
+                for image in repository.images:
+                    if (('imageDigest' in image_id and image.get_image_digest() == image_id['imageDigest']) or
+                            ('imageTag' in image_id and image.image_tag == image_id['imageTag'])):
+                        found = True
+                        response.add(image)
+                if not found:
+                    image_id_representation = "{imageDigest:'%s', imageTag:'%s'}" % (
```

Comment:
What do you think about using `json.dumps` here like in ecr/responses.py for crafting the JSON string? It might be easier to follow for future developers (I didn't immediately realize this was JSON at first)